src/py/flwr/common/message_test.py

```diff
@@ -62,12 +62,15 @@ def test_message_creation(
         if context:
             stack.enter_context(context)
 
-        _ = Message(
+        message = Message(
             metadata=metadata,
             content=None if content_fn is None else content_fn(maker),
             error=None if error_fn is None else error_fn(0),
         )
 
+        assert message.metadata.created_at > 0
```

Comment:
We could record the timestamp before the message gets created and assert that `created_at` is greater.